lib/greek_stemmer.rb

```diff
@@ -242,8 +242,14 @@ def stem(word)
       end
     end
 
+    # steb 6b
+    stem.scan(/^(.+?)(ΟΥΑ)$/ux) do |st, suffix|
```

Comment:
[minor] I don't think you need the parentheses around "OYA". Also, what is the `x` in the `/ux`? Typo?